lib.php

```diff
@@ -2173,6 +2244,46 @@ function plagiarism_urkund_send_files() {
     }
 }
 
+/**
+ * Function called by scheduled task to get files from Mahara.
+ *
+ */
+function plagiarism_urkund_fetch_mahara() {
```

Comment:
I'd be more inclined to drop this code into the task itself rather than a function within lib.php - nothing else will be calling this except the task right?